<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KBankAccountEditDecl</class>
 <widget class="QWidget" name="KBankAccountEditDecl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>191</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>-10</y>
     <width>294</width>
     <height>190</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::ExpandingFieldsGrow</enum>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>411</width>
     <height>182</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="transactionSystemLabel">
      <property name="text">
       <string>Transaction System:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="KComboBox" name="transactionSystemEdit">
        <property name="maxVisibleItems">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>National</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sepa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PayPal</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="KComboBox" name="countryCodeEdit">
        <property name="maxVisibleItems">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DE</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="bankAccountOwnerNameLabel">
      <property name="text">
       <string>Owner Name:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="KLineEdit" name="bankAccountOwnerNameEdit">
      <property name="inputMask">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="bankAccountNumberLabel">
      <property name="text">
       <string>Account No:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="KLineEdit" name="bankAccountNumberEdit">
      <property name="maxLength">
       <number>32767</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="bankCodeLabel">
      <property name="text">
       <string>Bank Code:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="KLineEdit" name="bankCodeEdit">
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="bankNameLabel">
      <property name="text">
       <string>Bank Name:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="KLineEdit" name="bankNameEdit">
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="accountNameLabel">
      <property name="text">
       <string>Account Name:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="KLineEdit" name="accountNameEdit">
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="defaultAccountEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="defaultAccountLabel">
      <property name="text">
       <string>Default Account:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
